locksmithctl/daemon.go

```diff
@@ -116,6 +117,52 @@ func expBackoff(interval time.Duration) time.Duration {
 	return interval
 }
 
+// getPeriodicAndSleep get the REBOOT_WINDOW_START, REBOOT_WINDOW_LENGTH
+// and check if everything is ok and the get then Periodic. Check if
+// it is outside of the reboot window and If yes then sleep.
+func getPeriodicAndSleep() error {
```

Comment:
we could change the name of this function to get a more meaningful sense from a high level point of view - we need to get a clear understand of what this function does by just reading its name.

What do you think of `waitForRebootWindow`  ?

Reply:
Yes `waitForRebootWindow` is more meaningful because our function is mainly for sleep and wait for next reboot window.